packages/snap/src/index.ts

```diff
@@ -384,6 +384,27 @@ export const onRpcRequest = async ({
       return await vcManager.getAllDecryptedVCs();
     }
 
+    case RPCMethods.ExportIdentity: {
+      const identityStorage = await getItemFromStore(StorageKeys.identity);
+
+      if (!identityStorage.privateKeyHex) {
+        throw new Error('Identity not created');
+      }
+
+      return snap.request({
+        method: 'snap_dialog',
+        params: {
+          type: 'alert',
+          content: panel([
+            heading('Your identity private key'),
```

Comment:
```suggestion
            heading('Your private key'),
```

Reply:
Wouldn't it be misunderstod by the users? whether it's identity or MM account PK?

Reply:
Fair point. How about `Your private RariMe key`?

Reply:
@ihordiachenko IMO `Your RariMe private key` or `Your identity private key` sounds better as we do not break `private key` phrase

Reply:
@lukachi change it to `Your RariMe private key`